src/Glob.php

```diff
@@ -61,7 +61,7 @@ protected static function systemGlob($pattern, $flags)
                 self::GLOB_NOSORT   => GLOB_NOSORT,
                 self::GLOB_NOCHECK  => GLOB_NOCHECK,
                 self::GLOB_NOESCAPE => GLOB_NOESCAPE,
-                self::GLOB_BRACE    => GLOB_BRACE,
+                self::GLOB_BRACE    => defined('GLOB_BRACE') ? GLOB_BRACE : 0,
```

Comment:
You're welcome. Wouldn't you please mind to notice me when the next release with this is launched? Because I've did that change on a *docker* container after running compose and I will need to remove it later.

Thanks.